tests/tensor_ops/test_sequence_ops.py

```diff
@@ -23,48 +21,42 @@ def test_input_validation(arrays):
         multiply_sequence(*arrays)
 
 
-def _broadcast_shapes(shape_a, shape_b):
-    longest, shortest = (
-        (shape_a, shape_b) if len(shape_a) >= len(shape_b) else (shape_b, shape_a)
-    )
-    shortest = (len(longest) - len(shortest)) * (1,) + shortest
-    return tuple(a if a > b else b for a, b in zip(longest, shortest))
+def prod(seq):
+    return reduce(lambda x, y: x * y, seq)
```

Comment:
https://docs.python.org/3/library/operator.html

Looks like you want `operator.mul`!